lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.rb

```diff
@@ -1,4 +1,5 @@
 require 'csv'
+require 'json-schema'
```

Comment:
Does this need to be in the gemspec?

Reply:
somehow I see json-schema is included in the ruby.
I think it, perhaps, is a sub-dependency from the tbd package.